app/web/src/components/Navigation/Navigation.tsx

```diff
@@ -1,22 +1,29 @@
 import {
   AppBar,
+  Avatar,
```

Comment:
Don't import from `@material-ui/core` for the `Avatar` since we have a component built on top of that we use across the app. Instead, import from `components/Avatar` instead